here-naksha-lib-psql/src/jvmMain/resources/naksha_sql_fixme.sql

```diff
@@ -0,0 +1,568 @@
+  let session = naksha.NakshaSession.Companion.get();
+  let mapi = require("jbon").Jb.map;
+  let map = mapi.newMap();
+  map["created_at"] = created_at;
+  map["updated_at"] = updated_at;
+  map["txn"] = txn;
+  map["action"] = action;
+  map["version"] = version;
+  map["author_ts"] = author_ts;
+  map["uid"] = uid;
+  map["puid"] = puid;
+  map["ptxn"] = ptxn;
+  map["app_id"] = app_id;
+  map["author"] = author;
+  map["geo_grid"] = geo_grid;
+  return session.xyzNsFromRow(collection_id, map)
+$$;
+
+CREATE OR REPLACE FUNCTION feature_to_jsonb(feature bytea) RETURNS jsonb
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  let jb = require("jbon");
+  let reader = new jb.JbMapFeature();
+  reader.mapBytes(feature);
+  let map = reader.root().toIMap();
+  map["id"] = reader.id();
+  return map;
+$$;
+
+CREATE OR REPLACE FUNCTION feature_to_json(feature bytea) RETURNS text
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  let jb = require("jbon");
+  let reader = new jb.JbMapFeature();
+  reader.mapBytes(feature);
+  let map = reader.root().toIMap();
+  map["id"] = reader.id();
+  return JSON.stringify(map);
+$$;
+
+CREATE OR REPLACE FUNCTION jsonb_to_feature(feature jsonb) RETURNS bytea
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  return require("jbon").JbBuilder.Companion.create(1000).buildFeatureFromMap(feature);
+$$;
+
+CREATE OR REPLACE FUNCTION json_to_feature(feature text) RETURNS bytea
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  return require("jbon").JbBuilder.Companion.create(1000).buildFeatureFromMap(JSON.parse(feature));
+$$;
+
+CREATE OR REPLACE FUNCTION feature_id(feature bytea) RETURNS text
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  let naksha = require("naksha");
+  let session = naksha.NakshaSession.Companion.get();
+  return session.getFeatureId(feature);
+$$;
+
+CREATE OR REPLACE FUNCTION feature_type(feature bytea) RETURNS text
+LANGUAGE 'plv8'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+  let naksha = require("naksha");
+  let session = naksha.NakshaSession.Companion.get();
+  return session.getFeatureType(feature);
+$$;
+
+CREATE OR REPLACE FUNCTION naksha_geometry_in_type(geo_type int4, geo_bytes bytea) RETURNS geometry
+LANGUAGE 'plpgsql'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+BEGIN
+  IF geo_type = 1 THEN
+    RETURN ST_GeomFromWKB(geo_bytes);
+  ELSIF geo_type = 2 THEN
+    RETURN ST_GeomFromEWKB(geo_bytes);
+  ELSIF geo_type = 3 THEN
+    RETURN ST_GeomFromTWKB(geo_bytes);
+  ELSE
+    RETURN null;
+  END IF;
+END;
+$$;
+
+CREATE OR REPLACE FUNCTION naksha_geometry(flags int, geo_bytes bytea) RETURNS geometry
+LANGUAGE 'plpgsql'
+IMMUTABLE
+PARALLEL SAFE
+SET search_path FROM CURRENT
+AS $$
+BEGIN
+  return naksha_geometry_in_type(naksha_geo_type(flags),geo_bytes);
+END;
+$$;
+
```

Comment:
How much of this "green" section was actually changed and how much is just the effect of weird GitHub diff colouring? ;)

Reply:
The fix-me file is basically just a backup of everything that seems to have no use, so github is right in that all of this is new, but we should not care about this file. We will delete it, I just kept the code around in case I missed something, while reducing the used naksha-sql.